src/tests/mocks/mock_agents.py

```
@@ -0,0 +1,30 @@
+from typing import Any
+from pydantic import BaseModel
```

Ruff (0.11.9)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)